### PR TITLE
Fix creating microphone session

### PR DESCRIPTION
**Pull request by anna1901**

### CODE REVIEW

File: AirCasting/CreateSessionViews/SelectDeviceView.swift [hunk 1]
```diff
@@ -108,6 +108,7 @@ struct SelectDeviceView: View {
             micLabels
         })
         .buttonStyle(WhiteSelectingButtonStyle(isSelected: selected == 2))
+        .disabled(microphoneManager.isRecording)
```
Comment (jankrzempek) [inline AirCasting/CreateSessionViews/SelectDeviceView.swift:111]: Maybe we could add and alert that can inform user about the thing that he can only make only one session and that's why it is grayed out?

**Review by jankrzempek — APPROVED** [review]
🥳🥳🥳

File: AirCasting/MicrophoneSession/MicrophoneManager.swift [hunk 2]
```diff
@@ -52,22 +52,22 @@ final class MicrophoneManager: NSObject, ObservableObject {
         try audioSession.setActive(true)
         recorder.isMeteringEnabled = true
         
-        createMeasurementStream(for: session) { result in
+        createMeasurementStream(for: session) { [unowned self] result in
             switch result {
-            case .success(let id):
-                self.measurementStreamLocalID = id
+            case .success():
```
Comment (pawelgil) [inline AirCasting/MicrophoneSession/MicrophoneManager.swift:57]: you don`t need `()` here

**Review by pawelgil — APPROVED** [review]
I think there might be some unsafe reads/writes from random threads to public properties in microphone manger, but this is something that was also present before this PR so I'll not include any remarks (I'm also going to refactor all this soon, so no point in fixing all that)

File: AirCasting/MicrophoneSession/MicrophoneManager.swift [hunk 3]
```diff
@@ -52,22 +52,22 @@ final class MicrophoneManager: NSObject, ObservableObject {
         try audioSession.setActive(true)
         recorder.isMeteringEnabled = true
         
-        createMeasurementStream(for: session) { result in
+        createMeasurementStream(for: session) { [unowned self] result in
```
[the remark this answers is not in the document]
Comment (anna1901) [reply AirCasting/MicrophoneSession/MicrophoneManager.swift:55]: Thanks, now I will remember about this for sure 😄

**Pull request merged**